database/migrations/2023_12_08_184652_mpls_addrtype_fix.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        DB::statement("ALTER TABLE mpls_sdps MODIFY COLUMN sdpFarEndInetAddressType ENUM('unknown', 'ipv4', 'ipv6', 'ipv4z', 'ipv6z', 'dns')");
```

Comment:
Hi @murrant 
Seems that I have an issue with this `enum` change. It is not supported on SQLite, meaning we are stuck. I could delete and recreate the column (but would loose the data if I do so). What do you think ?

Reply:
I would guess that if we recreate the column, the discovery/polling will update everything and would not be a problem.

Reply:
Changed the migration accordingly